docs/INSTALL_kafka_test_results.md

```diff
@@ -0,0 +1,424 @@
+
+# HYDRA Scaling Test 3: Apache Kafka
+
+## Test Goal
+
+The goal of this test is to stress the distributed messaging system **Apache Kafka** with increased client load. The general setup is quite simple. A Kafka Producer sends records at a--maximum--pre-defined transmission rate to the Kafka Server (i.e., cluster) which holds on to these records and hands them out to consumer(s). In this test we assume the following considerations:
+
```

Comment:
Does it make sense to mention about where every component is running. It would make it little easier to understand.
For now, I am assuming that broker, producer and consumer, all are running on single slave node of cluster.